ground/src/main/java/com/google/android/ground/model/geometry/Geometry.kt

```diff
@@ -92,8 +87,8 @@ data class LinearRing(val coordinates: List<Coordinate>) : Geometry {
    * Returns a *synthetic* coordinate containing the maximum x and y coordinate values of this ring.
    */
   private fun maximum(): Coordinate {
-    val maximumX = this.coordinates.maxOfOrNull { it.x }
-    val maximumY = this.coordinates.maxOfOrNull { it.y }
+    val maximumX = this.coordinates.maxOfOrNull { it.latitude }
```

Comment:
I would change these to use the new terminology lat/lng. Also, X should map to Lng and Y to Lat.